server/commands/VolumeCommand.ts

```diff
@@ -2,17 +2,40 @@ import CommandDefinition from '../domain/CommandDefinition';
 import IoConnection from '../clients/IoConnection';
 import VolumeEventMessage from "../domain/event-messages/VolumeEventMessage";
 
+const volChangeTypes: { [s: string]: string; } = {
```

Comment:
Prefer volume over vol.
```suggestion
const volumeChangeTypes: { [s: string]: string; } = {
```